ST_mortality.c

```diff
@@ -643,18 +717,28 @@ static void _slow_growth( const SppIndex sp) {
   Mem_Free(kills);
 }
 
-/***********************************************************/
+/**
+ * \brief Implements age-independent mortality.
+ * 
+ * Kills individuals in a species by the age-independent function (eqn 14) in C&L'90
+ * assuming that AGEMAX was defined.
+ * 
+ * Annuals are NOT killed here. Also, if Species[sp]->max_age == 0 this function 
+ * does nothing.
+ * 
+ * Initial programming by Chris Bennett @ LTER-CSU 6/15/2000  
+ * 
+ * \param sp The index of the species in the Species array.
+ * 
+ * \sideeffect The Species[sp]->IndivHead linked list might lose some individuals
+ *             if this function decides to kill them.\n
```

Comment:
this is strangely stated. I think it means some individuals may be removed from the linked list if killed?

Reply:
@kpalmqui I modified the comment in commit 984bd13. Let me know what you think!

Reply:
Much better, thanks!!